docs/pages/toolchain/sozo/world-commands/account.mdx

```diff
@@ -0,0 +1,73 @@
+import WorldOptions from '../common/world-options.mdx'
+import StarknetOptions from '../common/starknet-options.mdx'
+
+## Sozo Account 
+
+The `sozo account` command allows managing account contracts. 
+It provides functionalities to create new account configurations, deploying them, and fetch 
+account configuration for already existing account.
+
+Note that we currently only support the OpenZeppelin account variant. For more advanced use cases, consider using [starkli](https://book.starkli.rs/accounts).
+
+## Usage
+
+```bash
+sozo account [OPTIONS] <COMMAND>
+
+Commands:
+  new     Create a new account configuration without actually deploying.
+  deploy  Deploy account contract with a DeployAccount transaction.
+  fetch   Fetch account config from an already deployed account contract.
+  help    Print this message or the help of the given subcommand(s).
+
+```
+### Options
+
+- `-v, --verbose`: Increase logging verbosity.
+- `-q, --quiet`: Decrease logging verbosity.
+- `-h, --help`: Print help (see more with '--help').
+- `-V, --version`: Print version.
+
+### Subcommands
+
+#### `new`
+
+Create a new account configuration without actually deploying.
+
+- **Options:**
+  - `signer`: Options for specifying the signer.
+  - `force`: Overwrite the account config file if it already exists.
+  - `file`: Path to save the account config file.
+
+#### `deploy`
+
+Deploy account contract with a `DeployAccount` transaction.
+
+- **Options:**
+  - `starknet`: Options for specifying StarkNet configuration.
+  - `signer`: Options for specifying the signer.
```

Comment:
can you please add more description for options that are not directly arguments, something like: https://book.dojoengine.org/toolchain/sozo/world-commands/execute#world-options.

so user can know exactly what argument they need to pass in the CLI.